src/main/java/com/csemosip/bookingservice/controller/Bookings.java

```diff
@@ -49,12 +51,14 @@ public ResponseEntity<Map<String, Object>> findBookings(
     }
 
     @GetMapping("/{id}")
+    @PreAuthorize("hasAnyAuthority('ADMIN', 'RESOURCE_MANAGER', 'RESOURCE_USER')")
     public ResponseEntity<Map<String, Object>> findBookedResourcesById(@PathVariable Long id) {
         Booking booking = bookingService.findBookedResourcesById(id);
         return sendSuccessResponse(booking, HttpStatus.OK);
     }
 
     @PostMapping
+    @PreAuthorize("hasAnyAuthority('ADMIN', 'RESOURCE_USER')")
```

Comment:
Anyone should be able to create a booking, including a resource manager (they're not necessarily always resource managers)